modules/generative-ollama/clients/ollama_meta_test.go

```diff
@@ -0,0 +1,67 @@
+//                           _       _
```

Comment:
you can remove this file, or change the `metaInfo()` method to return the `Ollama` contents